### ISSUE TITLE
difference between cross_encoder and hf AutoModelForSequenceClassification

### ISSUE THREAD

**Issue by dctelus** [UKPLab/sentence-transformers#1568]
I see the cross_encoder module uses transformers AutoModelForSequenceClassification under the hood and was wondering what is the difference between them. I'm trying to figure out specifically what is different between the CrossEncoder.fit() and using the Trainer API to train that model. Is it the loss function? Because I see Bert models have the same loss function (e.g. BCEWithLogitsLoss) 

Any insight is helpful!

**Comment by nreimers** [comment]
No difference, just a wrapper which makes the usage nicer

**Comment by dctelus** [comment]
Gotcha, thx!
